pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -230,25 +230,46 @@ func (ds *dockerService) makeSandboxDockerConfig(c *runtimeApi.PodSandboxConfig,
 		// TODO: Check if this works with per-pod cgroups.
```

Comment:
Nit, you could extract a function here called `applyLinuxSpecificOptions` and the whole method would read as a transaction script on a uniform level of abstraction.